crypto/store/store_lib.c

```diff
@@ -231,6 +231,8 @@ int OSSL_STORE_close(OSSL_STORE_CTX *ctx)
 {
     int loader_ret;
 
+    if (ctx == NULL)
```

Comment:
This should be backported to 1.1.1 to avoid changing the semantics. So the callers can depend on the NULL check.

Reply:
PR for backport this enhancement of (allowing `NULL` arg for `OSSL_STORE_close()`) is in #11912.